src/main/java/com/amazon/opendistroforelasticsearch/sql/plugin/RestSqlAction.java

```diff
@@ -112,7 +112,8 @@ protected RestChannelConsumer prepareRequest(RestRequest request, NodeClient cli
             }
 
             final SqlRequest sqlRequest = SqlRequestFactory.getSqlRequest(request);
-            LOG.info("[{}] Incoming request {}: {}", LogUtils.getRequestId(), request.uri(), sqlRequest.getSql());
+            LOG.info("[{}] Incoming request {}: {}", LogUtils.getRequestId(), request.uri(),
+                    sqlRequest.removeSensitiveData());
```

Comment:
why adding a method on SqlRequest? could we use QueryDataMask.mask(sqlRequest.sql) instead.